docs/contribute/documentation.md

```diff
@@ -53,30 +66,21 @@ Example:
 * [Eclipse content of this page](?tab=eclipse)
 
 ## Compile GitHub Pages
-### Compile and Show via Docker
-The whole environment described below is already available in the docker image `starefossen/github-pages`. Use the following
```

Comment:
So this is now no longer a viable option? You have to have the required software installed locally?

If so, how much work would it be to replicate the old setup? I was actually quite a fan of only having to start the docker container and then being able to see the modified page locally, meaning I wouldn't have to deal with the build process.